src/components/Posts/ComposePost.tsx

```diff
@@ -96,7 +94,6 @@ const ComposePost = ({ placeholder }: ComposePostProps) => {
           </Box>
         </Box>
       </Box>
-      <Divider light />
```

Comment:
I think we should keep this divider, it helps differentiate the PostItem from the ComposePost component